### DISCUSSION TITLE
New name?

### DISCUSSION THREAD

**Discussion by MCJack123** [MCJack123/craftos2#49] [Ideas]
It has been suggested that I rename this project, but I'm not sure what it should be named. If you would like to suggest a new name, post it here.

**Comment by Qonsyuma** [comment]
Cactus / CactOS

**Comment by daelvn** [comment]
> Cactus / CactOS

Not an OS. I would be okay with Cactus. I guess. I don't know.

**Comment by Ocawesome101** [comment]
Ideas:

`CC: Terminal`

`CCTerminal`

`CCTerminal`

`CCTerm`

`CC: Desktop`

**Comment by Ocawesome101** [comment]
I'm not sure if this project needs a name besides its current one, tbh.
